help/view-and-edit-your-message-drafts.md

```diff
@@ -1,7 +1,8 @@
-# View and edit your message drafts
+# Message drafts
```

Comment:
Would "Draft messages" be a more appropriate title?

Reply:
Yeah, I think it would also be more consistent with the other titles in this section.